### PR TITLE
travis: add chihaya e2e testing

### LINKED ISSUE #151: Integrate 'Poke' tool into continuous integration
At the moment it's not tested at all

### PR DESCRIPTION
**Pull request by mrd0ll4r**
Example travis log [looks a little cluttered](https://travis-ci.org/chihaya/chihaya/jobs/324691145) but seems to work.

Fixes #151

### CODE REVIEW

**Comment by jzelinskie** [comment]
I wonder if this is something we should just vendor into the repository and name it like `cmd/chihaya-e2e`.

**Comment by mrd0ll4r** [comment]
I thought about that too, but then we have a direct dependency on `github.com/astaxie/torrent`, which to me looks a little weird, given that we have the `bittorrent` package which has type definitions for all the bittorrent-related things too. We just don't have a mechanism to perform announces.

**Comment by mrd0ll4r** [comment]
Alternative idea: What if we put it in github.com/chihaya/ekop ?

**Comment by jzelinskie** [comment]
tbh, i'm kinda freaked out by the fact that the bencode repo has to live there forever to make the builds for 1.0 reproducible...

**Comment by mrd0ll4r** [comment]
Oh, I never considered that. Alright, I'll move it to `cmd/chihaya-e2e` then

File: cmd/chihaya-e2e/main.go [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+import (
+	"crypto/rand"
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/anacrolix/torrent/tracker"
+	"github.com/pkg/errors"
```
Comment (jzelinskie) [inline cmd/chihaya-e2e/main.go]: you need to add these to dep

**Comment by mrd0ll4r** [comment]
alright, ptal

**Pull request merged**